src/components/NavBar.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <nav>
+    <div class="nav-container">
+      <ul class="links-container">
+        <h6>
+          <router-link to="/"
+            ><img src="@/assets/logo_green.png"
+          /></router-link>
+        </h6>
+        <h6>
+          <router-link to="/HelloWorld">HelloWorld</router-link>
+        </h6>
+        <h6>
+          <router-link to="/SearchResults">SearchResults</router-link>
+        </h6>
+        <h6>
+          <router-link to="/About">About</router-link>
+        </h6>
+        <h6>
+          <router-link to="/FAQ">FAQ</router-link>
+        </h6>
+      </ul>
+    </div>
+  </nav>
+</template>
+
+<style scoped>
+img {
+  height: 50px;
+}
+
+.nav-container {
+  margin: 0 10%;
```

Comment:
delete line 33 ```margin: 0 10%```

Note: If the margin property has two values:
```margin: 10px 5px;```
top and bottom margins are 10px
right and left margins are 5px